core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -509,11 +474,14 @@ protected ReadOnlyKeyCommand remoteReadOnlyCommand(InvocationContext ctx, ReadOn
       return new TxReadOnlyKeyCommand(command, getMutationsOnKey((TxInvocationContext) ctx, command.getKey()));
    }
 
+   protected RpcOptions createRpcOptions() {
+      return rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS_IGNORE_LEAVERS, DeliverOrder.NONE).build();
```

Comment:
Wouldn't it be better to store this as a protected variable in start() as the returned value is constant?